frontend/src/components/ImplementationReportView/DialectCompliance.tsx

```diff
@@ -42,7 +42,18 @@ const DialectCompliance: React.FC<{
               .map(([dialectName, result], index) => {
                 return (
                   <tr key={index}>
-                    <td>{Dialect.forPath(dialectName).uri}</td>
+                    <td>
+                      {Dialect.forPath(dialectName).uri}{" "}
```

Comment:
Please don't use spaces for positioning purposes - this should be handled with CSS styles

Reply:
Using margin mx-1 instead of spaces.